docs/src/filters.md

```diff
@@ -57,18 +57,24 @@ Most analog and digital filters are constructed by composing
 [response types](@ref response-types), which determine the frequency
 response of the filter, with [design methods](@ref design-methods),
 which determine how the filter is constructed.
+The response type is low-pass, high-pass, band-pass or
```

Comment:
Could you add links to the response types? i.e. instead of using "low-pass", use ```[`Lowpass`](@ref)```. More details on the link syntax [here](https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@ref-link-1).